WooCommerce/WooCommerceTests/ViewRelated/Search/Order/OrderSearchUICommandTests.swift

```diff
@@ -0,0 +1,171 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+import Storage
+import Networking
+import protocol WooFoundation.Analytics
+
+final class OrderSearchUICommandTests: XCTestCase {
+    let siteID: Int64 = 12345
+    private var storageManager: MockOrderStatusesStoresManager!
+    private var analyticsProvider: MockAnalyticsProvider!
+    private var analytics: WooAnalytics!
+
+    override func setUpWithError() throws {
+        super.setUp()
+        storageManager = MockOrderStatusesStoresManager()
+        analyticsProvider = MockAnalyticsProvider()
+        analytics = WooAnalytics(analyticsProvider: analyticsProvider)
+    }
+
+    override func tearDownWithError() throws {
+        storageManager = nil
+        analyticsProvider = nil
+        analytics = nil
+        super.tearDown()
+    }
+
+    func testCreateStarterViewControllerReturnsNil() throws {
+        // Given
+        let command = OrderSearchUICommand(siteID: siteID, onSelectSearchResult: { _, _ in }, storageManager: storageManager)
+
+        // When
+        let starterViewController = command.createStarterViewController()
+
+        // Then
+        XCTAssertNil(starterViewController, "Expected createStarterViewController to return nil")
```

Comment:
It's not clear _why_ this should be nil. When that's the case, I try to make it clearer in the tests e.g. in this comment, or the test name.
```suggestion
        XCTAssertNil(starterViewController, "Expected createStarterViewController to return nil so that the empty results table will be shown before the search.")
```

Reply:
Renamed in commit 1b2dc9f. Having not written the original type, I'm not sure about the intent either. My focus has been on verifying the implementation as is.